FlowCrypt/Functionality/Services/Key Services/KeyService.swift

```diff
@@ -100,19 +102,21 @@ final class KeyService: KeyServiceType {
             }
 
         // append keys to ensure with a pass phrase
-        if let passPhrase = passPhrase {
-            let keysToEnsure = keysInfo.map {
-                PrvKeyInfo(
-                    private: $0.private,
-                    longid: $0.primaryLongid,
-                    passphrase: passPhrase,
-                    fingerprints: Array($0.allFingerprints)
-                )
+        let keysToEnsure: [PrvKeyInfo] = keysInfo.compactMap {
```

Comment:
The original wording here was unclear. The comment should be removed, and `keysToEnsure` should be called `keysWithPassPhrases` - in another PR.